### DISCUSSION TITLE
This package or @nuxt/apollo?

### DISCUSSION THREAD

**Discussion by BorisKamp** [Diizzayy/nuxt-graphql-client#404] [Q&A]
Hi!

I have `"@nuxtjs/apollo": "^5.0.0-alpha.7` installed on my Nuxt3 package (nuxt `3.7.4`) but I just found out there's another package: this one.

Why are there two? I wonder because the author/owner seems to be the same person: @Diizzayy 

What is the difference between the two and what are the future plans for both?

**Comment by ehsanonline** [comment]
[nuxt-graphql-client](https://github.com/Diizzayy/nuxt-graphql-client) use [graphql-request](https://www.npmjs.com/package/graphql-request)
`@nuxtjs/apollo` use apollo graphql.
